setup.py

```diff
@@ -12,14 +12,15 @@
     version = f.read().strip()
 
 install_requires = [
+    'numpy>=1.16.4',
```

Comment:
Why does wazimap need numpy?

Reply:
I can't remember why this was added again but I will look into it. There were a lot of dependency issues while upgrading so it could have been due to a new package requiring this. Also I need to add all the updated packages from the requirements.txt into here.